samples/ext/nffs/prj.conf

```diff
@@ -0,0 +1,25 @@
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_SOC_FLASH_NRF5=y
+CONFIG_SOC_FLASH_NRF5_DEV_NAME="NRF5_FLASH"
```

Comment:
remove `CONFIG_SOC_FLASH_NRF5_DEV_NAME` - it was replaced by device tree property - so now you should use macro auto-generated FLASH_DEV_NAME for calls to nRF5 flash driver.